### ISSUE TITLE
OpenShot 2.4.0 Transitions not working in Linux Mint 18.2

### ISSUE THREAD

**Issue by scottydonline** [OpenShot/openshot-qt#1012]
**System Details: HP Elitebook with i5 M540, 4GB memory, NVIDIA GT218M
* Operating System / Distro: Linux Mint 18.2 Cinnamon 64-bit
* OpenShot Version: 2.4.0
* _Please attach log files if crash_

**Issue Description and steps to reproduce:**

Transitions not working at all between two clips. I get a black flash at the end of one clip, then the new clip shows a few seconds later. I have tried several methods to include a manual fade out on one clip and a manual fade in on the second and overlapping as well with no success. I have also tried the FADE transition overlapping between the two clips with the same result.

**Comment by DylanC** [comment]
@scottydonline 

![transition](https://user-images.githubusercontent.com/4488048/32878568-770c0192-ca9e-11e7-9174-723f29ef0d78.png)

Its working for me. Try to place the transition on top of the clip you want to use it on.

**Comment by scottydonline** [comment]
Sorry... I've tried it just like you suggested with no success. I remain with the same issue.

**System Details: HP Elitebook with i5 M540, 4GB memory, NVIDIA GT218M

Operating System / Distro: Linux Mint 18.2 Cinnamon 64-bit
OpenShot Version: 2.4.0
Please attach log files if crash
Issue Description and steps to reproduce:

Transitions not working at all between two clips. I get a black flash at the end of one clip, then the new clip shows a few seconds later. I have tried several methods to include a manual fade out on one clip and a manual fade in on the second and overlapping as well with no success. I have also tried the FADE transition overlapping between the two clips with the same result.

**Comment by DylanC** [comment]
@scottydonline 

Maybe try the v2.4.1 release. This should be working without any issue.

**Comment by scottydonline** [comment]
I tried v2.4.1 and experienced the same issue. The only stable version I have found that transitions work on Mint 18.2 is Openshot v1.4.3

**Comment by DylanC** [comment]
@scottydonline 
No problem. I will have to try this on Mint myself. On Ubuntu I can't see an issue with it.

**Comment by DylanC** [comment]
@scottydonline - Installed from a PPA?

**Comment by peanutbutterandcrackers** [comment]
@scottydonline - I had the same issue. I'm not sure if you did the same mistake, or not but the following solved it for me:

So, in openshot, transitions are supposed to go ON TOP OF the second clip IN the intersection. Like so:
![correct_transition_openshot](https://user-images.githubusercontent.com/26764547/36057169-220239cc-0e33-11e8-89f6-ec40c9e1287c.png)
Please note that the transition is placed at the exact start of the 2nd clip.]

And not so:
![incorrect1](https://user-images.githubusercontent.com/26764547/36057210-ab106edc-0e33-11e8-8b4e-f1bd0eb0c85a.png)
[ I know this is more intuitive. I used to do this all the time before I realized the correct way to place transitions ]

nor:
![incorrect2](https://user-images.githubusercontent.com/26764547/36057216-b82d608e-0e33-11e8-8bbf-9c19ed6091f1.png)

**Comment by peanutbutterandcrackers** [comment]
@scottydonline - Here's a way to know for sure if it is a bug in the program, or an inherent design decision: Try overlapping two clips. It will result in a transition being placed at the intersection, automatically. Test to see if the issue also manifests itself in the automatic transition. If not, this is not a fault in the program.

Hope this helps. :)

**Comment by IzzySoft** [comment]
I've got the same issue here on Mint 18.3: Transitions worked with 1.4.3 from Mints/Ubuntus repo, but don't work with 2.4.2 of the PPA or 2.4.3 from the AppImage. Auto transitions work, but I couldn't get manual transitions working. Same thing with 2s black somewhere short after its start (and yes, that's in the transition itself as the "blackout" was still there when separating it).

So according to the conclusion in the previous comment, that must be a user error. Whatever I tried, I failed. Except in v1.4.3. As I succeeded there, that should rule out the PEBKAC. So what's left now? Suggestions welcome.
